apps/cookbook/src/app/showcase/showcase.module.ts

```diff
@@ -1,23 +1,25 @@
-import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
 import { FormsModule } from '@angular/forms';
 import { IonicModule } from '@ionic/angular';
 
 import { KirbyModule } from '@kirbydesign/designsystem';
-import { COMPONENT_IMPORTS, COMPONENT_EXPORTS, COMPONENT_DECLARATIONS } from './showcase.common';
+
+import { IphoneModule } from '../iphone/iphone.module';
 import { CodeViewerComponent } from '../shared/code-viewer/code-viewer.component';
 import { ExampleViewerComponent } from '../shared/example-viewer/example-viewer.component';
 import { ShowcasePropertiesComponent } from '../shared/showcase-properties/showcase-properties.component';
-import { IphoneModule } from '../iphone/iphone.module';
+
 import { DividerShowcaseComponent } from './divider-showcase/divider-showcase.component';
+import { COMPONENT_DECLARATIONS, COMPONENT_EXPORTS, COMPONENT_IMPORTS } from './showcase.common';
 
 @NgModule({
   imports: [
     ...COMPONENT_IMPORTS,
     CommonModule,
     FormsModule,
     IonicModule,
-    KirbyModule,
+    KirbyModule.forChild({ moduleRootRoutePath: '/home/showcase/' }),
```

Comment:
Maybe loose the trailing slash? We should probably have a unit test for this anyhoo...